Assets/Scenes/SampleScene.unity

```diff
@@ -523,6 +523,9 @@ MonoBehaviour:
     additionalJumpHeightTime: 1
     canWallHang: 1
     wallHangTime: 2.5
+  square: {fileID: 0}
+  cat: {fileID: 0}
+  human: {fileID: 0}
```

Comment:
These need to be assigned values. You should get the box collider through code using `GetComponent`, and the cat and human sprites by dragging the sprite into the field in the editor. (The lack of values is also why the sprite disappears when you transition)